Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
-FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.4
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-debug-distroless-1.4
```

Comment:
No need to use the debug image. It was just a convenience for the performance tests.

hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4